lightTheme.ts

```diff
@@ -2,6 +2,20 @@ import { createTheme } from '@mui/material/styles';
 
 import { inter } from './constants/inter-latin';
 
+const color1 = '#141629';
+const color2 = '#4d4d4d';
+const color3 = '#fff';
+const color4 = '#2e8540';
+const color5 = '#eb5757';
+
+//const color6 = '#f3f3f3';
```

Comment:
Is it possible to remove that commented-out piece of code?

Reply:
'+